.github/workflows/brawlback.yml

```diff
@@ -0,0 +1,34 @@
+name: Brawlback Kit
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  BUILD_TYPE: Debug
```

Comment:
Is this BUILD_TYPE variable needed?

Cmake should use debug build type no matter what.

Reply:
It shouldn't be but let me check.